wordpress-mobile/WordPress-Aztec-iOS
# Allow editing of image properties on the DOM

This PR adds the following improvements:

 - Allow editing of image properties (align, size, src) on the DOM
 - Improve the image details properties screen
 - Some bug fixes around attachment

How to test:

 - Insert a image on the post using the media picker or the HTML view
 - Long press on the image
 - Change the image details
 - See if visually they are updated
 - Check the html node to see if the correspondent element attributes are updated
 - Edit the properties in HTML mode
 - Check if the new value of properties show visually and on the image details screen.
